src/Api/CMakeLists.txt

```diff
@@ -29,3 +31,14 @@ target_link_libraries(Api PUBLIC
         OrbitBase
         OrbitProducer
         ProducerSideChannel)
+endif()
+
+add_library(ApiInterface INTERFACE)
```

Comment:
What's the difference between this and `Api` above and what's the reason for having both? It's strange that they share files. It's possible that a common target can be extracted (for example, `Api` above can depend on this one).

Reply:
Fixed, Api depends on ApiInterface now. The "ApiInterface" is used if a project wants to define its own implementation, this is done so that introspection shares the same interface as the manual instrumentation API we ship to users.